src/parser/ast/ImplicitReturnStatementNode.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2016-present Samsung Electronics Co., Ltd
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ *  USA
+ */
+
+#ifndef ImplicitReturnStatementNode_h
+#define ImplicitReturnStatementNode_h
+
+#include "StatementNode.h"
+
+namespace Escargot {
+
+class ImplicitReturnStatementNode : public StatementNode {
+public:
+    explicit ImplicitReturnStatementNode(Node* argument)
+        : StatementNode()
+        , m_argument(argument)
+    {
+    }
+
+    virtual ASTNodeType type() override { return ASTNodeType::ImplicitReturnStatement; }
+    virtual void generateStatementByteCode(ByteCodeBlock* codeBlock, ByteCodeGenerateContext* context) override
+    {
+        ASSERT(context->tryCatchWithBlockStatementCount() == 0);
+#ifdef ESCARGOT_DEBUGGER
+        ASSERT(context->m_breakpointContext->m_lastBreakpointLineOffset == 0);
+        context->insertBreakpointAt(context->m_codeBlock->asInterpretedCodeBlock()->functionStart().line, this);
```

Comment:
How about merging this part into the original `ReturnStatementNode` instead of adding new `ImplicitReturnStatementNode`?
It seems that we can recognize each implicit return statement by checking `context->m_breakpointContext->m_lastBreakpointLineOffset == 0` or `codeBlock->currentCodeSize() == 0`.

Reply:
Good idea! That's what I've been looking for.